pkg/kotsclient/channel.go

```diff
@@ -168,9 +168,11 @@ func (c *GraphQLClient) CreateChannel(appID string, name string, description str
 }
 
 func (c *VendorV3Client) GetChannel(appID string, channelID string) (*channels.AppChannel, []channels.ChannelRelease, error) {
-	var response = KotsGetChannelData{}
+	response := KotsGetChannelData{}
 
-	url := fmt.Sprintf("/v3/app/%s/channel/%s", appID, channelID)
+	v := url.Values{}
+	v.Set("channelID", channelID)
+	url := fmt.Sprintf("/v3/app/%s/channel/%s", appID, v.Encode())
```

Comment:
url.Values are for arguments that come after `?` in the url.   In this case we should use `url.QueryEscape(channelID)`